pkg/cli/cmds/root.go

```diff
@@ -12,6 +13,13 @@ var (
 	debug bool
 )
 
+func init() {
+	// hack - force "file,dns" lookup order if go dns is used
+	if os.Getenv("RES_OPTIONS") == "" {
+		os.Setenv("RES_OPTIONS", " ")
```

Comment:
does this mean to make forceCgoLookupHost true?

Reply:
Yes, that's basically the intention.  If you know a better approach please let us know.  The idea here is that netgo doesn't properly read /etc/hosts because of the lack of nsswitch.conf.  So forcing to netcgo will use the compiled in musl behavior also which will do the proper thing.

Reply:
isn't the netcgo tag specified in the other patch enough?

Reply:
i think it was the intention of netcgo tag. doesn't it work?
https://github.com/golang/go/commit/b615ad8fd57f9394db14e403d12061c369379c52

Reply:
I think where possible we want to use netcgo, but if not possible or for dependencies which are built with netgo setting RES_OPTIONS will force the correct lookup order.